creator-node/test/lib/genericBullQueueMock.js

```diff
@@ -0,0 +1,40 @@
+const PrometheusRegistry = require('../../src/services/prometheusMonitoring/prometheusRegistry')
+const Bull = require('bull')
+
+const config = require('../../src/config')
+
+// Mock monitoring queue that sets monitor values on construction
+class GenericBullQueue {
+  constructor() {
+    this.queue = Bull('genericBullQueue', {
+      redis: {
+        host: config.get('redisHost'),
+        port: config.get('redisPort')
+      },
+      defaultJobOptions: {
+        removeOnComplete: 0,
+        removeOnFail: 0
+      }
+    })
+    const prometheusRegistry = new PrometheusRegistry()
+    prometheusRegistry.startQueueMetrics(this.queue)
+
+    this.queue.process(1, async (job) => {
+      console.log(this.params)
```

Comment:
actuallyyyyyy the more I think about it could we move this log 😬 kind of pointless hehe

Reply:
ya nbd